tests/core/test_actions_highlevel.py

```diff
@@ -1141,6 +1142,7 @@ def get_checkbox_elem(obs):
 
     obs, reward, term, trunc, info = env.step(action)
     checkbox = get_checkbox_elem(obs)
+    assert obs['mouse_position'] == [x, y]
```

Comment:
That's a good test, can you do the same for other pages which have iFrames, and check you get the correct coordinates when clicking on elements inside the iframe? (clicking with coordinates, and with bid)